app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Api::V1::UsersController < Api::ApiController
+  def index
+    @users = User.all
```

Comment:
it is bad to fetch all users, what if you will have 10000000 users, you will load all of them?

Reply:
I need list of all users for partial to show them on index page